datarobot_batch_scoring/main.py

```diff
@@ -262,7 +262,8 @@ def parse_generic_options(parsed_args):
     resume = parsed_args.get('resume')
     compression = parsed_args['compress']
     out_file = parsed_args['out']
-    timeout = int(parsed_args['timeout'])
+    timeout = parsed_args.get('timeout')
```

Comment:
(minor): `argparse` ensures it's `int` (if passed) or `None` (if not passed). Moreover, it will always exist within `parsed_args`. So we can just write `timeout = parsed_args['timeout']` without any feat here.

P.S: It's very opinionated, but I usually like programming by contracts since it helps to understand flow better: in which cases the value must be set or where it can be None.